src/SFML/Graphics/Font.cpp

```diff
@@ -53,8 +54,9 @@ namespace
 // FreeType callbacks that operate on a sf::InputStream
 unsigned long read(FT_Stream rec, unsigned long offset, unsigned char* buffer, unsigned long count)
 {
-    auto  convertedOffset = static_cast<sf::Int64>(offset);
-    auto* stream          = static_cast<sf::InputStream*>(rec->descriptor.pointer);
+    sf::Int64        convertedOffset = static_cast<sf::Int64>(offset);
```

Comment:
Any reason you've removed `auto` keyword here? There's also a place `GraphicsUtil.hpp` that you removed `auto`. I don't think these are ncesessary changes to make

Reply:
Don't even remember why I did that 😅

Reply:
Agreed. Reverted